host/resource/resource.go

```diff
@@ -33,6 +33,10 @@ const (
 	// practice, a 1000 milliCPU limit is equivalent to 1024 CPU shares.
 	TypeCPU Type = "cpu"
 
+	// TypeDisk specifies the available disk space in bytes inside the
+	// container.
+	TypeDisk = "disk"
```

Comment:
This should probably be `TypeWritableOverlayVolume` or `TypeTempDisk` or something along those lines so it doesn't get confused with the data volume support.